src/commands/utils/color.ts

```diff
@@ -1,6 +1,8 @@
 import { AttachmentBuilder, EmbedBuilder } from "discord.js";
 import sharp from "sharp";
 import { SlashCommand } from "../_types";
+// @ts-expect-error CJS
```

Comment:
no

Reply:
> no

do you want a module declaration or what

Reply:
Use a better package or just vendorize the whole thing

Reply:
> Use a better package or just vendorize the whole thing

... wdym by the second one

Reply:
all the packages are from before ESM so ig i could make my own but its just one comment

Reply:
Copy the colors into this codebase, from the package or wherever

Reply:
> Copy the colors into this codebase, from the package or wherever

i was going to do that but where do you want me to put them

Reply:
`lib` probably